--- apport-0.108/apport/report.py~	2008-04-01 17:02:29.000000000 +0300
+++ apport-0.108/apport/report.py	2008-04-24 13:33:28.907912824 +0300
@@ -208,7 +208,7 @@
         self['Package'] = '%s %s%s' % (package,
             packaging.get_version(package),
             self._pkg_modified_suffix(package))
-        self['SourcePackage'] = packaging.get_source(package)
+        self['RPMSourcePackage'] = packaging.get_source(package)
         self['PackageArchitecture'] = packaging.get_architecture(package)
 
         # get set of all transitive dependencies
