--- hardy/gtk/apport-gtk~	2008-05-17 12:54:36.000000000 +0200
+++ hardy/gtk/apport-gtk	2008-08-24 20:33:42.684738172 +0200
@@ -36,7 +36,7 @@
         # load UI
         gtk.window_set_default_icon_name("apport")
         gtk.glade.textdomain(self.gettext_domain)
-        self.widgets = gtk.glade.XML(os.path.join(os.path.dirname(sys.argv[0]),
+        self.widgets = gtk.glade.XML(os.path.join('/usr/share/apport',
             'apport-gtk.glade'))
         self.widgets.signal_autoconnect(self)
 
