--- ubuntu/backends/packaging_rpm.py~	2008-04-06 21:05:21.000000000 +0200
+++ ubuntu/backends/packaging_rpm.py	2008-04-06 21:15:26.310825923 +0200
@@ -52,7 +52,7 @@
         # parse this package's Requires
         reqs=[]
         for r in hdr['requires']:
-            if r.startswith('rpmlib'):
+            if r.startswith('rpmlib') or r.startswith('uname('):
                 continue # we've got rpmlib, thanks
             if r[0] == '/': # file requires
                 req_heads = self._get_headers_by_tag('basenames',r)
